webrtc-stats.html

```diff
@@ -1956,17 +2039,38 @@ <h2>
               </dt>
               <dd>
                 <p>
-                  Only valid for video. <code>fullFramesLost</code> is the cumulative number of
+                  The cumulative number of
                   full frames lost, as defined in Appendix A (i) of [[!RFC7004]].
                 </p>
               </dd>
+            </dl>
+          </section>
+        </div>
+      </section>
+      <section id="aststats-dict*">
+        <h3>
+          <dfn>RTCAudioStreamTrackStats</dfn> dictionary
+        </h3>
+        <div>
```

Comment:
t would be helpful to have a paragraph here saying this is the dictionary that RTC[Local/Remote]AudioStreamTrackStats inherits from, that there are no instances of this dictionary directly.

Reply:
Isn't this clear from the WebIDL? There's precedent here with e.g. RTCReceivedRTPStreamStats, so I recommend a separate issue.